components/tools/OmeroPy/test/integration/gatewaytest/test_get_objects.py

```diff
@@ -571,6 +571,33 @@ def testGetImage(self, gatewaywrapper, author_testimg_tiny):
         assert image.getOwnerOmeName == testImage.getOwnerOmeName
         assert image.getThumbVersion() is not None
 
+    @pytest.mark.parametrize("load_pixels", [True, False])
+    @pytest.mark.parametrize("load_channels", [True, False])
+    def testGetImageLoadPixels(self, load_pixels, load_channels,
+                               gatewaywrapper, author_testimg_tiny):
```

Comment:
Yes, but that will be a big change to the whole gatewaytest setup since it works in a very different way to our other tests. In fact, I would prefer that we completely get rid of the gateway test setup since it always tries to login as existing user and re-use existing data which fails if the data is not in expected state. It requires you to always try to clean up in every test so as not to corrupt data and you also don't always have the groups & permissions that you want etc.